.cbmc-batch/include/proof_helpers/make_common_data_structures.h

```diff
@@ -52,7 +52,8 @@ struct store_array_list_byte {
 bool is_valid_array_list(struct aws_array_list *list);
 
 /**
- * Checks whether the members of an aws_array_list structure are bounded by max_initial_item_allocation and max_item_size
+ * Checks whether the members of an aws_array_list structure are bounded by max_initial_item_allocation and
+ * max_item_size
```

Comment:
I would squash the clang-format commit into the previous one to yield a cleaner git history.